### DISCUSSION TITLE
How can I get packet loss ratio?

### DISCUSSION THREAD

**Discussion by Shohei-Takezawa** [USC-ACTLab/crazyswarm#617] [Q&A]
Hello,

I would like to investigate how the packet loss rate changes when:
- the number of aircraft is increased.
- there are external influences. (wi-fi, etc.)

If you know, please advise me on a good way to investigate the packet loss rate.

I think I can calculate the packet loss ratio if I can obtain the following:
- Number of packets sent by server
- Number of packets received by each cf

Thank you so much in advance.

**Comment by whoenig** [comment]
You could try `rosrun crazyflie_tools comCheck` at least for the external influences. The influence of more CFs is much harder, because lots of packets are send via broadcasts only. In that case, you might need some firmware changes to collect the statistics that you need as logging variables.
